src/main/java/ceos/backend/global/common/entity/University.java

```diff
@@ -0,0 +1,5 @@
+package ceos.backend.global.common.entity;
+
+public enum University {
+    SOGANG, YONSEI, EWHA, HONGIK
```

Comment:
`_INTERNAL_SERVER_ERROR(INTERNAL_SERVER_ERROR, "GLOBAL_500_1", "서버 오류. 관리자에게 문의 부탁드립니다.")`
에러 코드 이넘 일부인데
여기처럼 code를 입력했을 때, 그 코드에 해당하는 값을 꺼내올 수 있게 변경 가능한가요??

사실 지금 방법대로 해도 문제는 없습니다.
프론트에서 변환하라고 하면 되니까요 ㅎㅎ

```suggestion
    SOGANG("서강대학교"), ....
```